### ISSUE TITLE
Move Slack registration into the after-site-signup user flow, and ask users some CB Connect questions

### ISSUE THREAD

**Issue by lpatmo** [codebuddies/codebuddies#931]
h/t @angelocordon for this idea and drafting up the sample questions.

The idea:

1. Remove Slack signup/registration buttons from home page.
2. When a new user logs in for the first time using either Slack or Github, ask them some additional introductory questions on the "confirm your name" screen, like:
+ Where are you from? (gives us a sense of time zone)
+ What do you currently do? (ideas on professional background)
+ What do you want to do? (idea to connect them with someone who could align with this)
... and so on

NOTE: let's discuss in thread exactly what these questions should be. We could even have it be entirely CB Connect (https://bit.ly/codebuddies-connect) questions. 

3. User will receive an invite to the CodeBuddies Slack team (or, alternatively, simply see a button they can click on to invite themselves, if getting the invite to fire is tricky) only after they've submitted this welcome form.

4. Show this welcome form to existing users, too, if they haven't filled out their data.

5. Show data on user's profile page, but allow certain answers to be kept private (LET'S DISCUSS)

5. Note: We might also want to build an introductory welcome email into this user flow. This could be a separate issue/PR.

## Thoughts
- Perhaps asking more "About You" questions upfront will deter folks who join the Slack to simply do promotion w/out contributing to the community
- Asking people upfront what they're looking for can help us connect them immediately with possible study partners/mentors/mentees, etc. (See: CB Connect project)
- Downside of implementing this: barrier to join Slack, but upside: know better people's backgrounds and what they're looking for
- Again, let's finalize what questions we want to display in thread

**Comment by lpatmo** [comment]
## Design Concept #1: 

Screen 1/2:
![image](https://user-images.githubusercontent.com/4512699/43562405-30be89a4-95d1-11e8-9243-d6b9ffac7051.png)

Screen 2/2: 
![image](https://user-images.githubusercontent.com/4512699/43562718-190c4696-95d3-11e8-9b94-6284c476338b.png)

Second screen could probably be improved -- eg maybe split up into further steps?? Show screenshots of Slack and the forum?? Walk through how hangouts and study groups work? Serve as a reference that users can toggle back to, maybe? Right now it's a summary.

**Comment by railsstudent** [comment]
tag @distalx

**Comment by lpatmo** [comment]
I'm going to separate out this issue into smaller issues. Closing this thread for now.
